docker/test/integration/cluster/containers/OPCUAServerContainer.py

```diff
@@ -31,7 +31,7 @@ def deploy(self):
 
         logging.info('Creating and running OPC UA server docker container...')
         self.client.containers.run(
-            "lordgamez/open62541:1.3.3",
+            "lordgamez/open62541:1.3.5",
```

Comment:
is it possible to host this under the apache user?

Reply:
I'm not sure as it is not an Apache product. I created an [issue](https://github.com/open62541/open62541/issues/5082) a while back to publish the open62541 for the new releases, but so far it has not been done yet.